python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -949,6 +984,101 @@ def calculateMetrics(self, science, difference, diaSources, kernelSources):
                 raise BadSubtractionError(ratio=metrics.differenceFootprintRatioStdev,
                                           threshold=self.config.badSubtractionVariationThreshold)
 
+    def filterSatellites(self, diaSources, science):
+
+        wcs = science.getWcs()
+        nbbox = []
+
+        for source in diaSources:
+            fp = source.getFootprint()
+            source_bbox = fp.getBBox()
+
+            corners = [wcs.pixelToSky(source_bbox.beginX, source_bbox.beginY),
+                       wcs.pixelToSky(source_bbox.beginX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.beginY)]
+
+            tmp = []
+            for c, corner in enumerate(corners):
+                tmp.append([corner.getRa().asDegrees(), corner.getDec().asDegrees()])
+            nbbox.append(tmp)
+
+        detector_id = science.getDetector().getId()
+        visit_id = science.getInfo().getVisitInfo().getId()
+
+        dia_sources_json = []
+        for i, source in enumerate(diaSources):
+            dia_sources_json.append(
+                {"diasource_id": source['id'], "bbox": nbbox[i]})
+
+        sattle_output = requests.put(
+            f'{self.config.sattle_host}:{self.config.sattle_port}/diasource_allow_list',
+            json={"visit_id": visit_id, "detector_id": detector_id, "diasources": dia_sources_json})
```

Comment:
Outside the scope of this review, but PUT doesn't seem like the right method for a query. Should this be GET/POST instead?